tests/integration/security/reachability_test.go

```diff
@@ -123,11 +127,26 @@ func TestReachability(t *testing.T) {
 							return opts.Target == rctx.Naked || (opts.Target == rctx.B && opts.PortName != "http")
 
 						}
+						// Headless service with sidecar injected, autoMtls does not apply.
```

Comment:
s/headless with sidecar injected, global mTLS enabled, no client side transport socket or transport_socket_matches since it's headless service.

Reply:
can you address this, by updating the comments?